src/server/components/notebook-actions-menu.jsx

```diff
@@ -8,9 +8,9 @@ import DeleteModal from "./delete-modal";
 import UploadModal from "./upload-modal";
 import {
   selectFileAndFormatMetadata,
-  uploadFile,
-  deleteNotebookOnServer
+  uploadFile
 } from "../../shared/utils/file-operations";
+import { deleteNotebookRequest } from "../../shared/server-api/notebook";
 
 export default class NotebookActionsMenu extends React.Component {
```

Comment:
the `uploadFile` method already returns json, so the `NotebookActionMenu` throws an error on file upload

Reply:
Oops, right.